distributions/demo-google-deployment/transfer/build.gradle

```diff
@@ -26,29 +45,69 @@ repositories {
 
 dependencies {
     compile project(':portability-transfer')
-
-    compile project(':extensions:data-transfer:portability-data-transfer-flickr')
+    compile project(':extensions:config:portability-config-yaml')
+    // TODO: depend on these based on list in flag values.
     compile project(':extensions:data-transfer:portability-data-transfer-google')
+    compile project(':extensions:data-transfer:portability-data-transfer-instagram')
     compile project(':extensions:data-transfer:portability-data-transfer-microsoft')
-    compile project(':extensions:cloud:portability-cloud-google')
-    compile project(':extensions:config:portability-config-yaml')
+    compile project(':extensions:data-transfer:portability-data-transfer-flickr')
+    compile project(':extensions:data-transfer:portability-data-transfer-rememberthemilk')
 }
 
 addCloudExtensionDependency(project)
```

Comment:
Should we explicitly include the Google Cloud extension here or leave it up to the flag? Does it ever make sense to build this with any other cloud type?

Reply:
I actually brought this up in Slack too, that this should always be cloudType Google. cloudType is referenced other places in the main build file, so I thought it would be better to set cloudType to Google since we will need that anyway, and allow addCloudExtensionDependency to use that. That also allows for addCloudExtensionDependency to be refactored to a common place and not need to hardcode the cloud extension for each distro.

Unfortunately, setting cloudType is harder than it seems, Jim is going to come back to that in his PR, but for now we're just setting it on the command line to ./gradlew

Reply:
sgtm, optionally if its possible, can we check that the cloudType in the param is Google and fail the build?

Reply:
that's a good idea but I think it may fail the Travis build depending how it is done. Since Jim is coming right back to this I left a TODO, since it will always be google in his impl.